code/Data exploration.R

```diff
@@ -175,16 +200,46 @@ data %>% filter(is.na(TEMPERATURE))
 data %>% filter(is.na(INITIAL_FORECAST))
 data %>% filter(is.na(FINAL_FORECAST))
 
-############################
-##DERIVE ADDITIONAL FIELDS##
-############################
+################################
+##Aggregate the data to hourly##
+################################
 
-##convert DATETIME to date -- no longer required, reading in as right field on auto
-##data$DATETIME = ymd_hms(data$DATETIME);
+#Since the forecast made by models will be hourly
 
 ##create year/month/day/hour/minute fields
-data <- data %>% mutate(YEAR=year(DATETIME), MONTH=month(DATETIME),DAY=day(DATETIME));
-data <- data %>% mutate(HOUR=hour(DATETIME), MINUTE=minute(DATETIME));
+data <- data %>% mutate(YEAR=year(DATETIME), 
```

Comment:
[question]: what are we using these fields for? to derive extra features?

Reply:
I used it for filtering purpose (particularly for charts)